lib/core/utils/get-selector.js

```diff
@@ -41,7 +41,11 @@ function getAttributeNameValue(node, at) {
 				return;
```

Comment:
At L39 above, is it worth doing `escapeSelector(value)` as well?

Reply:
Probably would be a good idea as well